checkov/argo_workflows/runner.py

```diff
@@ -128,14 +128,12 @@ def extract_image(self, file_path: str, container: dict[str, Any]) -> Image | No
         if image_name and isinstance(image_name, str):
             start_line = container.get("__startline__", 0)
             end_line = container.get("__endline__", 0)
-            image_id = self.inspect(image_name)
-            if image_id:
-                return Image(
-                    file_path=file_path,
-                    name=image_name,
-                    image_id=image_id,
-                    start_line=start_line,
-                    end_line=end_line,
-                )
+            return Image(
+                file_path=file_path,
+                name=image_name,
+                image_id="",
```

Comment:
I noticed all `Image` construction pass in `""` to `image_id`, do we update it on a later stage? or is that information not relevant to "named" images (python3.8:alpine for example)?
If the latter is true, I wanted to suggest creating a separate class for it / accepting `""` by default in the constructor so that we don't have to repeat the keyword argument

Reply:
good catch 💪 I left initially, because I thought it will be needed for some scenario, but actually it is not needed in the class at all. It is retrieved directly when needed.